Makefile

```diff
@@ -1,31 +1,27 @@
-help: ## Display this help screen
-	@grep -h -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-15s\033[0m %s\n", $$1, $$2}'
+.PHONY: wasm contracts keys circuits abi test run help
 
-wasm: ## Generate the WASM for the contract given (e.g. make wasm for=transfer)
-	@cargo rustc \
-		--manifest-path=contracts/$(for)/Cargo.toml \
-		--release \
-		--target wasm32-unknown-unknown \
-		-- -C link-args=-s
+wasm: ## Generate the WASM for all the contracts
+	$(MAKE) -C ./contracts/transfer/ wasm
+	$(MAKE) -C ./contracts/bid/ wasm
 
-contracts: ## Generate the WASM for all the contracts & test them
-	$(MAKE) -C ./contracts/transfer/ test
-	$(MAKE) -C ./contracts/bid/ test
+abi: ## Build the ABI and test it
+	$(MAKE) -C ./rusk-abi test
 
 keys: ## Create the keys for the circuits
 	cd rusk && cargo test --release -vv -- --nocapture
```

Comment:
This should be updated as well.
First of all, it doesn't generate the keys anymore since it doesn't have the environment variable set. Second, it seems to me that is "testing" Rusk and the keys generation is a side effect.